project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC13")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.20")
```

Comment:
Are you sure scalajs cross-build is necessary? I have a feeling it may complicate our build for questionable value.

Reply:
Let's discuss on the associated ticket.